datacube_alchemist/transforms.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict
+
+import numpy
+from datacube import Datacube
+from datacube.virtual import Measurement, Transformation
+from xarray import Dataset, merge
+
+
+class DeltaNBR(Transformation):
+    """Return NBR"""
```

Comment:
This should describe what it does, what can be configured, what it works on and what it doesn't...